working/augmentation-libraries/feature-specification.md

```diff
@@ -338,65 +355,67 @@ occur inside an augmenting type-introducing declaration, e.g., an
 augmenting class or mixin declaration). This is also true if `augmented()`
 occurs inside a local function declaration inside the body of that function
 or method declaration. We could say that `augmented` is a contextual
-keyword because it is unable to refer to a declaration in scope when it
-occurs inside an augmenting declaration, it always has the special meaning
-which is associated with augmentations.*
+reserved word because it is unable to refer to a declaration in scope when
+it occurs inside an augmenting declaration, it always has the special
+meaning which is associated with augmentations.*
 
 A compile-time error occurs if a declaration with the basename `augmented`
-occurs in a location where the outermost enclosing declaration is
-augmenting. *This error is applicable to all such declarations, e.g.,
-local functions, local variables, parameters, and type parameters.*
+occurs in a location where any enclosing declaration is augmenting. *This
+error is applicable to all such declarations, e.g., local functions, local
+variables, parameters, and type parameters.*
 
-A compile-time error occurs if the identifier `augmented` occurs in a
-non-augmenting declaration, of a kind that can be augmenting, inside an
-augmenting declaration.
+Consider a non-augmenting member declaration _Dm_ that occurs inside an
+augmenting type declaration _Dt_. A compile-time error occurs if the
+identifier `augmented` occurs in _Dm_.
 
 *For example, inside `augment class C` we could have a declaration like
-`void f() {...augmented()...}`.
-This is an error because the outer `augment` forces the meaning of `augmented`
-to be about augmentation in the entire scope, but the method declaration is an
-introduction, not an augmentation.*
+`void f() {...augmented()...}`. This is an error because the outer
+`augment` forces the meaning of `augmented` to be about augmentation in the
+entire scope, but the method declaration is introductory and hence there is
+no earlier declaration to augment.*
```

Comment:
Nice clarification of these two paragraphs.

Reply:
Thx!